### DISCUSSION TITLE
Graphs not updating

### DISCUSSION THREAD

**Discussion by antonmedv** [upptime/upptime#196] [Q&A] [answered]
Hi,

Why are graphs not updating? https://github.com/deployphp/status Even if trigger action automatically. 

Thanks.

**Comment by AnandChowdhary** [comment, marked as answer]
Seems like graphs were last updated 8 hours ago: https://github.com/deployphp/status/commit/70682cf347c48c818ea0fb6eb21c0caccd7d5a09. Since then, there is no change in the response time data, so there is no new data to update the graphs to. When you manually trigger the Graphs CI workflow, it just regenerates the same graph so there is nothing to commit.

By default, response time data is collected every day, so graphs are also regenerated once every day. If you want to do this more frequently, you can update the workflow crons: https://upptime.js.org/docs/configuration#ci-schedule.
